### ISSUE TITLE
`activeEditorState` is not appended to `window.title`

### ISSUE THREAD

**Issue by ntrogh** [microsoft/vscode#241848]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: Latest insiders
- OS Version: Windows latest

Testing TPI https://github.com/microsoft/vscode/issues/241740

Steps to Reproduce:

1. Toggle Screen Reader Accessibility Mode to enable it
2. Window title doesn't show the editor info.
3. The `window.title` setting doesn't have the `activeEditorState` included
4. Manually add `activeEditorState` to `window.title` setting shows the editor info in the window title as expected. 

https://github.com/user-attachments/assets/a7f24741-0179-4f22-9127-b9e8b770a83f

**Comment by meganrogge** [comment]
@ntrogh did you also try enabling a screen reader - like NVDA?

**Comment by ntrogh** [comment]
@meganrogge not yet - just set the screen reader mode on

**Comment by meganrogge** [comment]
Meaning do not modify the setting after enabling screen reader mode, just switch from the tab with edits then back to it. I believe the edits will be shown in the title.

**Comment by ntrogh** [comment]
I enabled screen reader mode, and then switched tabs, but nothing happens.

**Comment by meganrogge** [comment]
A gif would help because I cannot reproduce that on mac - when I enable screen reader mode, switch tabs, come back to a tab with decorations, I see that in the window title.

**Comment by ntrogh** [comment]
1. enable screen reader mode
2. Ctrl+tab to switch between tabs
3. Nothing happens to the title

https://github.com/user-attachments/assets/38f40c7f-1795-445a-8b32-60054365bdec

**Comment by meganrogge** [comment]
and you see the screen reader optimized status bar item?

**Comment by ntrogh** [comment]
Setting is enabled, and yes I see the Status Bar item (also shows in the video)

![Image](https://github.com/user-attachments/assets/bd04f2a6-0eaf-4a30-b2d2-c5353216405b)

**Comment by meganrogge** [comment]
Note #242082 when verifying
